### ISSUE TITLE
VPX 10.8.4 GL makes playfield dissapear when F1 is pressed.

### ISSUE THREAD

**Issue by mbp2728** [vpinball/vpinball#993]
**Describe the bug**
When i press f1, the table dissappear, can no longer see the playfield and the only way is to close VPX and re-start again.
I tested VPX 10.8.4 DX and GL versions. Vpx DX version does not have this issue. It only happens on VPX 10.8.4 GL.

**To Reproduce**
1. Start vpx 10.8.4 GL
2. Open and run any table with rom
3. Press f1
4. Table will dissapear and the window for f1 menu won't show up either

**Expected behavior**
F1 menu must show and playfield should be on the screen as well.

**Screenshots**
NA

**Crash 
NA

**Versions & Settings**
 - VPX: 10.8.4 latest version
 - Windows version 10
 - GPU and video settings if applicable
RTX4060
 - If this bug report is for VPX 10.8+, please join your setting file (locate 'VPinballX.ini' in '%AppData%\VPinballX', either zip it or change its extension to '.log', then drag'n drop it here)
 I'll send this week.
 - If this bug report is for VPX 10.8+, please enable log in your settingd and join your log file

**Additional context**

**Comment by Pittyh** [comment]
I had this bug too, i thought it was my setup.

**Comment by dacstyle** [comment]
Yes this happen to me too, if you press F1 you will get a black screen and need to do alt-tab in order to get the playfield again, then the vpinmame will appear.

**Comment by mbp2728** [comment]
Hello,

No, when using GL version, if i press ALT-TAB the playfield does not show
up again. The behavior you described sometimes happen on DX version. For DX
version the ALT-TAB is a good workaround. For the GL version, I have to
close VPX and restart it again, because I cannot close the table normally.

Em dom., 5 de nov. de 2023 às 19:18, dacstyle ***@***.***>
escreveu:

**Comment by dacstyle** [comment]
I'm using GL versión 64 days every day since months ago and I can see the playfield and menú after doing alt-tab

Are you in fullscreen mode? If yes, try again with windowed mode.

**Comment by mbp2728** [comment]
Ah, I don't use it in windowed mode. I'll try when I return home.

Thanks for the tip.

Em dom., 5 de nov. de 2023 às 23:37, dacstyle ***@***.***>
escreveu:

**Comment by dacstyle** [comment]
Hello @vbousquet I see this report was labelwd as incomplete, but there is no much more information. Just downloaded VPinballX_GL-10.8.1-2499-7cccf7f-Release-win-x64 
Enabled BAM Headtracking and opened a tabled. 
It looks bad, so I want to open the Headtracking menu in order to adjust the pov and the headtracking menu is not working.

I'm attaching two videos doing the same thing on 10.8 and 10.8.1 so you can see the difference,  if need any other information just let me know, I'm here for help.

As always a very big "thank you" for all the VPX team, you are bringing happiness to a lot of people all around the world.

https://github.com/user-attachments/assets/231ff8bb-473d-4d5f-bbc2-15744906b129


https://github.com/user-attachments/assets/21bb5113-6bd7-4aaa-94f9-0da9c1ced9a5
